tsconfig.json

```diff
@@ -8,19 +8,19 @@
     // "allowJs": true,                       /* Allow javascript files to be compiled. */
     // "checkJs": true,                       /* Report errors in .js files. */
     // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
-    "declaration": true,                      /* Generates corresponding '.d.ts' file. */
-    // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
-    // "sourceMap": true,                     /* Generates corresponding '.map' file. */
-    // "outFile": "./",                       /* Concatenate and emit output to single file. */
+    "declaration": true,                      /* Generates corresponding '.d.ts' fileName. */
```

Comment:
Hey I guess you unknowingly modified this file. All the `file` keyword is changed to `filename`. It is just in comments so doesn't matter.

Reply:
you're absolutely right, i assumed that was a thing that typescript did itself, but my search & replace went nuts. i'll quickly revert that!